lib/modules/manager/types.ts

```diff
@@ -21,6 +21,7 @@ export interface ExtractConfig extends CustomExtractConfig {
   npmrc?: string;
   npmrcMerge?: boolean;
   skipInstalls?: boolean | null;
+  repository?: string;
```

Comment:
interesting, is it really passed? 🤔

Reply:
Yeah, the `ExtractConfig` that gets passed to the managers is actually an instance of the `WorkerConfig` which has `repository?: string` as part of its inherited interface. So by adding it to the `ExtractConfig`, it gets mapped and made available to the manager.